src/app/home/pages/WelcomePage/WelcomePage.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import {
+  PageSection,
+  Flex,
+  FlexItem,
+  Text,
+  TextContent,
+  TextVariants,
+  Button,
+  Checkbox,
+} from '@patternfly/react-core';
+import logoMA from '../../../common/components/logoMA.svg';
+import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
+import flex from '@patternfly/react-styles/css/utilities/Flex/flex';
+import { IReduxState } from '../../../../reducers';
+import { AuthActions } from '../../../auth/duck/actions';
+import { history } from '../../../../helpers';
+const styles = require('./WelcomePage.module');
+
+interface IWelcomePageBaseProps {
+  isHideWelcomeScreen: boolean;
+  setWelcomeScreenBool: (value) => void;
+  setNamespaceSelectIsOpen: (value) => void;
+}
+
+// TODO each of these pages flashes the empty state while loading, we should show a loading spinner instead somehow.
+
+const WelcomePageBase: React.FunctionComponent<IWelcomePageBaseProps> = ({
+  isHideWelcomeScreen,
+  setWelcomeScreenBool,
+  setNamespaceSelectIsOpen,
+}: IWelcomePageBaseProps) => {
+  const onHandleGetStarted = (e) => {
+    const LS_KEY_ACTIVE_NAMESPACE = 'activeNamespace';
+    const activeNamespace = localStorage.getItem(LS_KEY_ACTIVE_NAMESPACE);
+    if (activeNamespace) {
+      history.push('/clusters');
+    } else {
+      setNamespaceSelectIsOpen(true);
+    }
+  };
+
+  return (
+    <>
+      <PageSection>
+        <Flex
+          className={`
+            ${spacing.mtXl} 
+            ${spacing.mbXl} 
+            ${flex.alignContentCenter} 
+            ${flex.justifyContentCenter} 
+            ${flex.alignItemsCenter} 
+            ${flex.flexDirectionColumn} 
```

Comment:
As awesome as these flex and spacing utility classes are, we can just wrap the whole thing in a `<Bullseye>` to center everything on the page, and then all we need here is `alignItemsCenter` and `flexDirectionColumn`.

Reply:
Resolved in 61262224ee126fc999bc0406107188376026ba3d